src/App.js

```diff
@@ -1,5 +1,99 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import { validateCarNameList, validateTryCount } from "./validation.js";
+import {
+  CAR_NAME_INPUT_MESSAGE,
+  TRY_COUNT_INPUT_MESSAGE,
+} from "./constants/inputMessage.js";
+import {
+  EMPTY_MESSAGE,
+  EXECUTE_RESULT_MESSAGE,
+  WINNER_MESSAGE,
+} from "./constants/outputMessage.js";
+
 class App {
-  async run() {}
+  #tryCount;
+  #cars;
+
+  constructor() {
+    this.#tryCount = 0;
+    this.#cars = [];
+  }
+
+  async run() {
+    let carNameList = await this.readCarName();
+    validateCarNameList(carNameList);
+    this.generateCars(carNameList);
+
+    const tryCount = await this.readTryCount();
+    validateTryCount(Number(tryCount));
+    this.#tryCount = Number(tryCount);
```

Comment:
<유저가 입력한 시도횟수>와 <유효성 검사가 끝난 값>이
구분되도록 변수명을 바꾸면 실수를 줄일 수 있지 않을까요? 😊

```suggestion
    const inputTryCount = await this.readTryCount();
    validateTryCount(Number(inputTryCount));
    this.#tryCount = Number(inputTryCount);
```

Reply:
오호 좋은 지적이세요..!
헷갈림 방지하기 위해 동일한 이름을 사용하는 방법을 택했는데, 오히려 이 방법이 실수를 초래할 수 있음을 깨닫게 되었습니다.
감사합니다 😄⭐